### PR TITLE
added resposive attribute on client side

### PR DESCRIPTION
**Pull request by asad-rafter**

### CODE REVIEW

**Review by yuval-cloudinary — CHANGES_REQUESTED** [review]
Please add the analytics tags

**Comment by yuval-cloudinary** [comment]
I now see the delay you mentioned. The right approach would be to hide or destory the gallery's container when leaving the page and show or create it when when entering a new page. Due to other concerns regards the gallery's implementation, let's postpone the work here (also the JIRA was postponed)

File: Cloudinary-pwa-kit/packages/pwa-cloudinary/app/components/cloudinary-lineItem-image/index.jsx [hunk 1]
```diff
@@ -11,16 +13,22 @@ const CloudinaryLineItemImage = ({ cldProduct = {}, image = {} }) => {
...
                     </AspectRatio>
                 </Box>
             ) : (
                 <>
                     {cldProduct?.url && (
                         <Image
+                            className={cldProduct?.isResponsive && 'cld-responsive'}
                             alt={image.alt}
-                            src={`${cldProduct.url}`}
-                            ignoreFallback={true}
```
Comment (yuval-cloudinary) [inline Cloudinary-pwa-kit/packages/pwa-cloudinary/app/components/cloudinary-lineItem-image/index.jsx:24]: why was this line removed? (I don't know if it is needed)

Comment (asad-rafter) [reply Cloudinary-pwa-kit/packages/pwa-cloudinary/app/components/cloudinary-lineItem-image/index.jsx:24]: Yeah, its not needed.

**Review by yuval-cloudinary — APPROVED** [review]
